modules/bound-sa-tokens-configuring.adoc

```diff
@@ -111,12 +111,20 @@ kind: Pod
 metadata:
   name: nginx
 spec:
+  securityContext:
+    runAsNonRoot: true
```

Comment:
Can we add an explanation for the `securityContext` configurations as done for the `serviceAccountName` below? This will help clarify the purpose and importance setting. Below are a few suggestions for reference:

`runAsNonRoot: true <1>`: Prevents containers from running as root to minimize compromise risks.

`seccompProfile: type: RuntimeDefault <2>`: Sets default Seccomp profile, limiting to essential system calls, reducing risks.

---

These explanations `<1>` and `<2>` are aimed at providing a clear understanding of the security measures being implemented within the pod's configuration. WDYT

Reply:
Thanks, updated!